src/loader.rs

```diff
@@ -485,7 +498,7 @@ impl<'b, 'w> Loader<'b, 'w> {
             // Either copy the already-created handle or create a new asset
             // for each image or texture to load.
             let handle = if let Some(handle) = self.scene.textures.get(&handle_label) {
-                trace!("Already encountered texture: {label}, skipping");
+                warn!("Already encountered texture: {label}, skipping");
```

Comment:
```suggestion
                debug!("Already encountered texture: {label}, skipping");
```

This shouldn't be a warn, it's expected. We cache the textures to avoid spending time reloading them